src/net.cpp

```diff
@@ -2687,13 +2689,20 @@ void RelayTransaction(const CTransactionRef &ptx, const bool fRespend)
         mapRelay.insert(std::make_pair(inv, ptx));
         vRelayExpiration.push_back(std::make_pair(GetTime() + 15 * 60, inv));
     }
+
     LOCK(cs_vNodes);
     for (CNode *pnode : vNodes)
     {
         if (!pnode->fRelayTxes)
         {
             continue;
         }
+        // If the transaction won't be acceptable to the target node, then don't send it.  This avoids the poisoning
+        // the node against this tx (via the node's alreadyHave() logic), so that it can be sent later.
```

Comment:
typo:

-This avoids the poisoning the node
+This avoids poisoning the node

Reply:
done